### PR TITLE
Show board

### PR DESCRIPTION
**Pull request by lindyrp**
* Add routes in config.rb
* Add show.html.erb with a visual representation of a chessboard, along with the related CSS file
* In `games_controller.rb`: delete `before_action :initialize_board!` -- this is handled as an after_action with create action elsewhere, also fix a typo in the params
* Bonus action! Adds a JS layer in `pieces.js` to collect coordinate data when clicking on board

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
   before_action :authenticate_user!
-  before_action :initialize_board!
+  #before_action :initialize_board!
+
+  def new
+    @game = Game.new
+  end
 
   def create
     game = Game.create(game_params)
```
Comment (ruishine) [inline app/controllers/games_controller.rb]: Should it be:
@game = Game.create(game_params)
?

File: app/assets/javascripts/pieces.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+$(document).ready(function(){
+  $("td").click(function(){
+    var data= $(this).data();
+    console.log("data: ", data);
+  });
+});
```
Comment (ruishine) [inline app/assets/javascripts/pieces.js]: excited to see this execute!

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (lindyrp) [inline app/controllers/games_controller.rb]: Oops, sorry I missed that! Thanks for catching.

**Comment by ilyakrasnov** [comment]
Looks good but seems to have some merge conflicts. Please check

**Comment by ilyakrasnov** [comment]
👍  Very well done - awesome idea to iterate twice!

**Comment by ruishine** [comment]
second thumbs up!

On Thu, Jun 23, 2016 at 11:01 AM, Ilya Krasnov <notifications@github.com>
wrote:

> 👍 Very well done - awesome idea to iterate twice!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Methodologists/metchess/pull/14#issuecomment-228098077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALOK7HvFrPkp0gcaCbLqy7wIh3fuEUBdks5qOq3jgaJpZM4I8S7f>
> .
>



-- 
-Lin Rui Li

**Comment by AlekstheRealist** [comment]
👍

**Pull request merged**